dipy/viz/colormap.py

```diff
@@ -255,3 +261,59 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
+    """Create colors from a specific colormap and return it
+    as an array of shape (N,3) where every row gives the corresponding
+    r,g,b value. The colormaps we use are similar with those of matplotlib.
+
+    Parameters
+    ----------
+    v : (N,) array
+        vector of values to be mapped in RGB colors according to colormap
+    name : str.
+        Name of the colormap. Currently implemented: 'jet', 'blues',
+        'accent', 'bone' and matplotlib colormaps if you have matplotlib
+        installed. For example, we suggest using 'plasma', 'viridis' or
+        'inferno'. 'jet' is popular but can be often misleading and we will
+        deprecate it the future.
+    auto : bool,
+        if auto is True then v is interpolated to [0, 10] from v.min()
+        to v.max()
+
+    Notes
+    -----
+    Dipy supports a few colormaps for those who do not use Matplotlib, for
+    more colormaps consider downloading Matplotlib (see matplotlib.org).
+    """
+
+    if name == 'jet':
+        msg = 'Jet is a popular colormap but can often be misleading and'
+        msg += 'we will remove it from being the default in the near future.'
+        msg += 'Try for example plasma, viridis, hot or inferno.'
+        print(msg)
```

Comment:
I think the direction is right. We want to warn people of this now, and deprecate later. You probably want to raise a warning here, rather than just printing a message, though.